ci/actions-templates/linux-builds-template.yaml

```diff
@@ -152,6 +163,27 @@ jobs:
           AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
           AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
         if: github.event_name == 'push' && github.ref == 'refs/heads/stable'
+      - name: Build a snapcraft configuration file
+        run: |
+          sh ci/snapcraft.sh
+        if: matrix.snap_arch != ''
+        env:
+          DO_SNAP: 1
+          SNAP_ARCH: ${{ matrix.snap_arch }}
+          SNAP_EDGE: 1 # skip-stable
+      - uses: jhenstridge/snapcraft-build-action@v1
+        id: snapcraftbuild
+        with:
+          path: snapcraft
+        if: matrix.snap_arch != ''
+      - uses: jhenstridge/snapcraft-publish-action@0dac40c79fd10f8f51ee149a58a47ab3a42aeee0
+        with:
+          store_login: ${{ secrets.SNAPCRAFT_TOKEN }}
+          snap: ${{ steps.snapcraftbuild.outputs.snap }}
+          release: edge # skip-pr skip-stable
+          release: beta # skip-pr skip-master
```

Comment:
This should just be:

    release: edge,beta

I don't think Github actions supports repeating input parameters for actions: they're passed to the action as environment variables rather than anything more structured.

Reply:
Those comments are directives to a script which generates our real workflow files, on PRs, there'll be no `release`, on master builds it'll say `edge` and on stable builds `beta`